### PR TITLE
Chore/update about us first section

### PR DESCRIPTION
**Pull request by klezi10**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |
|  [**121**](https://github.com/MarianaSouza/web-dev-path/issues/121) |

#### Have you updated the CHANGELOG.md file? If not, please do it.
Yes.

#### What is this change?
Changed the first section of "about us" page. The paragraph content stayed the same but the titles changed to 'version control' and 'peer reviews'. Added "our background" section.

#### Were there any complications while making this change?
No.

#### How did you verify this change?
Locally.

#### When should this be merged?
After review.

### CODE REVIEW

**Review by cherylli — COMMENTED** [review]
Looks good. I would remove all the commented lines of the old code and I think Title is no longer being used anywhere else, so it can be removed

**Comment by klezi10** [comment]
> I think Title is no longer being used anywhere else, so it can be removed

Do you mean remove the import in the about component or remove the js component entirely?

**Comment by cherylli** [comment]
> > I think Title is no longer being used anywhere else, so it can be removed
> 
> @cherylli Do you mean remove the import in the about component or remove the js component entirely?

I think remove the whole component if it's not in use. It's a very simple component anyway if we need it in future we can make it again.

**Review by mariana-caldas — APPROVED** [review]
Such a great job, @klezi10 !
All yours to merge after another reviewer approves it! 🚀

File: components/containers/RevealContentContainer.js [hunk 1]
```diff
@@ -15,7 +15,14 @@ const RevealContentContainer = ({ children }) => {
   }, [entry.isIntersecting]);
 
   return (
-    <div ref={ref} className={`${styles.wrapper} ${styles[hiddenStyle]}`}>
+    <div
+      ref={ref}
+      className={
+        hiddenStyle
```
Comment (cherylli) [inline components/containers/RevealContentContainer.js:21]: what's the purpose of this change? looks more complicated (harder to read).

**Review by cherylli — COMMENTED** [review]
I don't quite understand the CSS changes? Look a bit harder to comprehend. Maybe it's just me.

File: components/containers/Container.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-export default function Container({ className, children, styles }) {
+export default function Container({ customClass, children, styles }) {
   return (
     <div
-      className={`${containerStyles.container} ${className ? className : ''}`}
+      className={
+        customClass
+          ? `${containerStyles.container} ${customClass}`
+          : containerStyles.container
+      }
```
Comment (klezi10) [inline components/containers/Container.js:10]: Hey @mariana-caldas would you mind helping me understand this piece of code? I read it as -> if there is a custom class, to use the container styles first but override some of those properties with custom class styles?

Is ``customInnerClass='our-background'`` from about.js an example of a custom class?

File: components/containers/TwoColumn.js [hunk 3]
```diff
@@ -28,7 +28,11 @@ export default function TwoColumn({
   return (
     <section className={styles.wrapper} style={styleProps.wrapper}>
       <Container
-        className={`${styles.inner} ${styles[customInnerClass]}`}
+        customClass={
+          customInnerClass
+            ? `${styles.inner} ${styles[customInnerClass]}`
+            : styles.inner
```
Comment (klezi10) [inline components/containers/TwoColumn.js:34]: I'm having trouble understanding `${styles[hiddenStyle]}` when you have a chance please

**Comment by klezi10** [comment]
There is a bug with the footer - the alignment of the contents is off. 

I can easily change the nav to center in mobile size but I can't seem to re-align it for tablet and desktop size. I can only change it in container.module.scss where I add:

`.container {
...,
  display: flex;
  flex-direction: row;
  justify-content: space-between;
}`

but then it changes other parts of the website that are using container.

I will take a look again later but if anyone has any ideas, I would appreciate the help.

File: components/containers/Container.js (hunk 2 above)
Comment (mariana-caldas) [reply components/containers/Container.js:10]: Exactly. Those ternaries remove the "undefined" class behavior when no custom classes exist.

File: components/containers/TwoColumn.js (hunk 3 above)
Comment (mariana-caldas) [reply components/containers/TwoColumn.js:34]: Those ternaries prevent or remove the "undefined" class behavior when no custom classes exist.

**Comment by mariana-caldas** [comment]
> There is a bug with the footer - the alignment of the contents is off.
> 
> I can easily change the nav to center in mobile size but I can't seem to re-align it for tablet and desktop size. I can only change it in container.module.scss where I add:
> 
> `.container { ..., display: flex; flex-direction: row; justify-content: space-between; }`
> 
> but then it changes other parts of the website that are using container.
> 
> I will take a look again later but if anyone has any ideas, I would appreciate the help.

Hey @klezi10 :)
I can't see any problem with the footer elements alignment so far.  Could you please record us a loom video showing the problems you mention above in detail? If we see those are relevant, we can fix them in a second PR.
For now,  if you have no further questions, I think we should merge this feature. Thanks so much! 🚀

**Comment by klezi10** [comment]
Thanks @mariana-caldas once I get one more approve, I will merge.

Just to be safe here are the screenshots of the footer but it seems like it only appears on my end:

desktop:
desktop

small screen:
mobile

**Comment by mariana-caldas** [comment]
> Thanks @mariana-caldas once I get one more approve, I will merge.
> 
> Just to be safe here are the screenshots of the footer but it seems like it only appears on my end:
> 
> desktop: desktop
> 
> small screen: mobile

Thanks for the screenshots, @klezi10 ! That bug was created by me who forgot to change the Container custom class name property both on Newsletter and Footer. 🙌🏼

**Review by cherylli — APPROVED** [review]
